config_example.json

```diff
@@ -1,7 +1,5 @@
 {
   "__comment":"Example config, please copy and paste this as config.json",
-  "prefix": "..",
-  "__comment":"TODO feature",
-
-  "token": "Your Discord bot accounts token",
+  "prefix": "Enter the prefix you wish to use, such as --",
+  "token": "Enter your User Token, you can find that by searching 'Discord User Token'"
```

Comment:
just use "false"
they should know how to get their own token already if they wish to use this